modules/authentication/src/handlers/local.ts

```diff
@@ -81,6 +81,7 @@ export class LocalHandlers implements IAuthenticationStrategy {
         {
           path: '/forgot-password',
           action: ConduitRouteActions.POST,
+          description: `Used when user forgets password. Sends an email for password reset.`,
```

Comment:
That would more accurately be something like: `Resets the authenticated user's password. Sends a verification email if email verification is enabled AND verification emails are enabled AND the Email module available.`
That's a mouthful and still doesn't actually explain everything there is to be explained.
We respond with an appropriate message about verification being required, therefore we should probably keep it simple:

`Resets the authenticated user's password.`

To be honest we shouldn't be mentioning conditional config options at all unless enabled, just like for security clients, but propagating that info down here would be kinda meh unless done in a seemless way.

Reply:
Either way, this endpoint is exposed only when sendVerificationEmail config is enabled, so we could be a little more specific in the description I guess?

Reply:
Fair enough `Generates a password reset token and forwards a verification link to the user's email address.`